Signal/translations/en.lproj/Localizable.strings

```diff
@@ -783,6 +783,9 @@
 /* No comment provided by engineer. */
 "NETWORK_ERROR_RECOVERY" = "Please check you're online and try again.";
 
+/* Indicates to the user that censorship circumvention has been activated. */
+"NETWORK_STATUS_CENSORSHIP_CIRCUMVENTION_ACTIVE" = "Censorship Circumvention: On";
```

Comment:
@NicolasCGN Hmm, well it's not a button, it's a state indicator.  Also I'd like to be consistent and unfortunately the language we're using elsewhere is "censorship circumvention".